server/sonar-server/src/main/java/org/sonar/server/component/ws/ResourcesWs.java

```diff
@@ -44,11 +44,14 @@ private void defineIndexAction(NewController controller) {
       .setDescription("Gets a list of components. Requires Browse permission on resource.<br>" +
         "The web service is deprecated and you're invited to use the alternatives: " +
         "<ul>" +
-        "<li>if you need components only: api/components/tree</li>" +
-        "<li>if you need components with measures: api/measures/component_tree</li>" +
+        "<li>if you need one component without measures: api/components/show</li>" +
+        "<li>if you need one component with measures: api/measures/component</li>" +
+        "<li>if you need several components without measures: api/components/tree</li>" +
+        "<li>if you need several components with measures: api/measures/component_tree</li>" +
         "</ul>" +
-        "When you provide one metric, the number of results is limited to 500. When several metrics are provided, the number of results is not limited. " +
-        "This is a known limitation and it won't be fixed.")
+        "When you provide one metric, the number of results is limited to 500. When several metrics are provided, the number of measures is limited to 10000. " +
```

Comment:
As components are limited to 500, limiting measures is not required, especially because 10'000 means 200 measures per component, which is very high.